test-cases/longevity/longevity-large-partition-8h.yaml

```diff
@@ -15,10 +15,10 @@ prepare_verify_cmd: ["scylla-bench -workload=sequential -mode=read -replication-
                      "scylla-bench -workload=sequential -mode=read -replication-factor=3 -partition-count=25 -clustering-row-count=5555 -partition-offset=76 -clustering-row-size=uniform:1024..2048 -concurrency=100 -connection-count=100 -consistency-level=quorum -rows-per-request=10 -timeout=30s -iterations 1 -validate-data"
                     ]
 
-stress_cmd: ["scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=25 -clustering-row-count=10000 -partition-offset=401 -clustering-row-size=uniform:10..1024     -concurrency=10  -connection-count=10  -consistency-level=quorum -rows-per-request=10 -timeout=30s -retry-number=30 -retry-interval=80ms,1s -iterations 10",
-             "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=25 -clustering-row-count=100   -partition-offset=426 -clustering-row-size=uniform:2048..5120   -concurrency=10  -connection-count=10  -consistency-level=quorum -rows-per-request=10 -timeout=30s -retry-number=30 -retry-interval=80ms,1s -iterations 10",
-             "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=25 -clustering-row-count=10    -partition-offset=451 -clustering-row-size=uniform:5120..8192   -concurrency=10  -connection-count=10  -consistency-level=quorum -rows-per-request=10 -timeout=30s -retry-number=30 -retry-interval=80ms,1s -iterations 10",
-             "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=25 -clustering-row-count=1     -partition-offset=476 -clustering-row-size=uniform:8192..10240  -concurrency=10  -connection-count=10  -consistency-level=quorum -rows-per-request=10 -timeout=30s -retry-number=30 -retry-interval=80ms,1s -iterations 10",
+stress_cmd: ["scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=25 -clustering-row-count=10000 -partition-offset=401 -clustering-row-size=uniform:10..1024     -concurrency=10  -connection-count=10  -consistency-level=quorum -rows-per-request=10 -timeout=30s -retry-number=30 -retry-interval=80ms,1s -iterations 0 -duration=540m",
```

Comment:
This case actually shows the newly introduced limitation.
I remember that in some cases we have different durations for different stress commands.
And with this change we won't be able to set explicit different durations. Right?

Reply:
Yes, you right. for common case, all stress commands will be run with same stress duration. 
For specific case, when stress commands should be run with different time( why? ) this will not work, and it should be run like previously without setting stress_duration parameter

Reply:
> ( why? )

https://github.com/scylladb/scylla-cluster-tests/pull/6080#discussion_r1192468836

Reply:
@vponomaryov it's really not clear why anyone would want not all the stress command to be with the same duration.
it's kind of contradicts what longevity are usually for, on other cases like feature specific we might still do those kind of things.